apisix/init.lua

```diff
@@ -513,6 +513,8 @@ function _M.http_access_phase()
                 api_ctx.consumer,
                 api_ctx
             )
+            core.log.debug("find consumer ", api_ctx.consumer.username,
```

Comment:
IMHO, we can use `info` level error log, therefore we can check it without `log_level('debug')`. `debug` is too low to check it in the production if needed.

Reply:
Sure, just thought we don't want to clutter the log too much in a prod setting where you might be running with `info` level.